scenarios/generate-synthetic-data/ai-generated-data-query-response/generate-data-query-response.ipynb

```diff
@@ -147,16 +142,95 @@
-    "from azure.identity import DefaultAzureCredential\n",
     "\n",
-    "simulator = Simulator(azure_ai_project=azure_ai_project, credential=DefaultAzureCredential())"
+    "simulator = Simulator(model_config=model_config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data Source for the simulator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Wikipedia article\n",
+    "In this example we use a wikipedia article as raw text generate Query Response pairs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import wikipedia\n",
+    "\n",
+    "search_term = \"Leonardo da vinci\"\n",
+    "wiki_title = wikipedia.search(search_term)[0]\n",
+    "wiki_page = wikipedia.page(wiki_title)\n",
+    "text = wiki_page.summary[:5000]"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Connecting the simulator to your application"
+    "### Using data from an Azure AI Search Index\n",
+    "Use this step if you want to obtain text from your search index. This part assumes that you have a search index and have populated the index with necessary data that is relevant to your application."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "\n",
+    "search_term = \"camping stove\"\n",
+    "endpoint = \"https://<your-search-service-name>.search.windows.net\"\n",
+    "index_name = \"<your-index-name>\"\n",
+    "api_key = \"<your-api-key>\"\n",
+    "url = f\"{endpoint}/indexes/{index_name}/docs/search?api-version=2024-07-01\"\n",
+    "\n",
+    "headers = {\n",
+    "    \"api-key\": api_key,\n",
+    "    \"Content-Type\": \"application/json\",\n",
+    "}\n",
+    "search_query = {\"search\": search_term, \"top\": 1}\n",
+    "\n",
+    "response = requests.post(url, headers=headers, data=json.dumps(search_query))\n",
+    "if response.status_code == 200:\n",
+    "    results = response.json()\n",
+    "    for result in results[\"value\"]:\n",
+    "        text = result[\"content\"]\n",
+    "\n",
+    "text = text[:5000]  # lets make sure the text isn't too long to exceed the model's token limit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Connecting the simulator to your application\n",
+    "\n",
+    "This part assumes that you application is a call to `AzureOpenAI`'s chat completion endpoint. It also assumes that the following values are set in the environment. Feel free to change this method to call your application with its configuration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ[\"AZURE_OPENAI_API_KEY\"] = \"<your-api-key>\"\n",
```

Comment:
Can we move this section up to the Parameters cell and then reference the environment variables throughout instead of setting those values explicitly in the Parameters cell and then having this section below?

Then, users can know to set all of their environment variables and then run the notebook from there without any other configuration in later steps.

Reply:
@slister1001 will address this in her PR - https://github.com/Azure-Samples/azureai-samples/pull/138
I'm abandoning this PR once she has all the components from here included in hers